x/oracle/keeper/task.go

```diff
@@ -202,6 +205,9 @@ func (k Keeper) CreateTask(ctx sdk.Context, creator sdk.AccAddress, task types.T
 		if err := k.CollectBounty(ctx, task.GetBounty(), creator); err != nil {
 			return err
 		}
+		if _, ok := task.(*types.TxTask); ok {
+			k.CheckShortcutQuorum(ctx, task)
```

Comment:
The task can be added in SetClosingBlockStore and SetShortcutTasks?

Reply:
It shouldn't. Before `SetShortcutTasks`, `ShouldAgg` will be called to make sure the task is not eligible for `ClosingBlockStore`.

Reply:
ClosingBlockStore will be called firstly in CreateTask